inference-engine/src/preprocessing/ie_preprocess_data.cpp

```diff
@@ -814,8 +822,8 @@ void PreProcessData::execute(Blob::Ptr &outBlob, const PreProcessInfo& info, boo
                              "there's nothing to be done";
     }
 
-    if (_roiBlob == nullptr) {
-        THROW_IE_EXCEPTION << "Input pre-processing is called without ROI blob set";
+    if (inBlob == nullptr) {
+        THROW_IE_EXCEPTION << "Input pre-processing is called with null inBlob";
     }
 
     batchSize = PreprocEngine::getCorrectBatchSize(batchSize, _roiBlob);
```

Comment:
I suppose `_roiBlob` should be replaced with `inBlob` everywhere in the function body (including its sub-routines).

Reply:
not relevant any more - patch is reworked